campusmap/src/app/App.tsx

```diff
@@ -15,6 +18,12 @@ const App = (): ReactElement => {
           <Route path="/search">
             <SearchResultsPage />
           </Route>
+          <Route path="/info">
+            <InfoPage />
+          </Route>
+          <Route path="/*">
```

Comment:
This path doesn't seem to work properly. It's routing everything to the not found page.